new/aws-ec2-manager-blueprint.yaml

```diff
@@ -0,0 +1,675 @@
+  # Manager.
+  # #####################################################################
+  manager_configuration:
+    type: cloudify.nodes.MyCloudifyManager
+    properties:
+      ssh_user: { get_input: ssh_user }
+      ssh_key_filename: { get_input: ssh_key_filename }
+      agent_local_key_path: { get_input: agent_private_key_path }      
+            
+      cloudify:
+        plugins:
+          aws_config_path: /etc/cloudify/aws_plugin/boto
+
+        transient_deployment_workers_mode:
+          enabled: false
+          global_parallel_executions_limit: 50
+
+        cloudify_agent:
+          min_workers: 0
+          max_workers: 5
+          remote_execution_port: 22
+          user: { get_input: agents_user }
+          env:
+            AWS_CONFIG_PATH: { get_property: [SELF, cloudify, plugins, aws_config_path] }
+
+          
+        workflows:
+          task_retries: -1  # this means forever
+          task_retry_interval: 30
+
+        policy_engine:
+          start_timeout: 30
+
+    interfaces:
+      cloudify.interfaces.lifecycle:
+        configure:
+          implementation: fabric.fabric_plugin.tasks.run_task
+          inputs:
+            tasks_file: components/manager/scripts/aws-ec2/configure.py
+            task_name: configure_manager            
+            task_properties:
+              aws_config: { get_property: [ aws_configuration, aws_config ] }
+              manager_config_path: { get_property: [SELF, cloudify, plugins, aws_config_path] }
+
+            fabric_env:
+              user: { get_input: ssh_user }
+              key_filename: { get_property: [management_keypair, private_key_path] }
+              host_string: { get_attribute: [manager_server_ip, aws_resource_id] }
+        create:
+          implementation: fabric.fabric_plugin.tasks.run_script
+          inputs:
+            script_path: components/manager/scripts/configure_manager.sh
+            fabric_env:
+              user: { get_input: ssh_user }
+              key_filename: { get_property: [management_keypair, private_key_path] }
+              host_string: { get_attribute: [manager_server_ip, aws_resource_id] }
+              always_use_pty: true
+
+    relationships:
+      - type: cloudify.relationships.contained_in
+        target: manager_host
+        target_interfaces:
+          cloudify.interfaces.relationship_lifecycle:
+            postconfigure:
+             implementation: components/manager/scripts/set_manager_public_ip.sh
+             inputs:
+              public_ip: { get_attribute: [manager_server_ip, aws_resource_id] }
+      - type: cloudify.relationships.depends_on
+        target: nginx
+        target_interfaces:
+          cloudify.interfaces.relationship_lifecycle:
+            postconfigure: components/manager/scripts/set_manager_private_ip.sh
+
+
+  management_keypair:
+    type: cloudify.aws.nodes.KeyPair
+    properties:
+      use_external_resource: { get_input: use_existing_manager_keypair }
```

Comment:
I think you can use the same referencing mechanism as in fabric_env while overriding the differentiating field(s)